src/content/reference/es/p5/draw.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: draw
+module: Structure
+submodule: Structure
+file: src/core/main.js
+description: >
```

Comment:
This one is hard to read if directly translated into Spanish. So, I suggest the following structure: 

```html
    <p>
      La función <code>draw()</code> es la función central de un sketch. Se
      ejecuta de forma continua después de la función
      <a href="/reference/p5/setup">setup()</a>, hasta que el programa se
      detenga o se llame a la función <code>noLoop()</code>. La frecuencia de
      ejecución se establece por defecto en 60 fotogramas por segundo, y podemos
      ajustarlo mediante la función
      <a href="/reference/p5/frameRate">frameRate()</a>.
    </p>

    <p>
      Podemos controlar cuándo y cómo se ejecuta <code>draw()</code> con las
      funciones:
    </p>

    <ul>
      <li>
        <a href="/reference/p5/noLoop">noLoop()</a> - Detiene la ejecución de
        cíclica de <code>draw()</code>.
      </li>
      <li>
        <a href="/reference/p5/loop">loop()</a> - Reanuda la ejecución de
        cíclica <code>draw()</code>.
      </li>
      <li>
        <a href="/reference/p5/redraw">redraw()</a> - Ejecuta
        <code>draw()</code> una sola vez después de haber sido invocada.
      </li>
    </ul>

    <p>
      Nota: si se llama la función <code>noLoop()</code> dentro de <code>setup()</code>,
      <code>draw()</code> se ejecutará una vez más antes de detenerse.
    </p>

    <p>Consideraciones importantes:</p>
    <ul>
      <li>Solo puede haber una función <code>draw()</code> por sketch.</li>
      <li>
        La función <code>draw()</code> es indispensable para generar animaciones
        o procesar eventos como
        <a href="/reference/p5/mousePressed">mousePressed()</a>.
      </li>
      <li>
        Los estilos aplicados, como
        <code>fill()</code> y <code>stroke()</code>, se mantienen entre
        ejecuciones consecutivas de <code>draw()</code>.
      </li>
      <li>
        Al inicio de cada ejecución de <code>draw()</code>, el sistema de
        coordenadas se reinicia. Las transformaciones aplicadas previamente,
        como escalar, rotar y traslación, se eliminan evitando su acumulación.
      </li>
    </ul>
```

Reply:
these look good